src/main/java/uk/gov/hmcts/darts/arm/api/impl/ArmDataManagementApiImpl.java

```diff
@@ -43,14 +43,14 @@ public List<String> listResponseBlobs(String prefix) {
 
     public BinaryData getBlobData(String blobPathAndName) {
         return armService.getBlobData(
-                armDataManagementConfiguration.getContainerName(),
+            armDataManagementConfiguration.getContainerName(),
             blobPathAndName
         );
     }
 
     @Override
-    public void deleteResponseBlob(String blobName) {
-        armService.deleteResponseBlob(armDataManagementConfiguration.getContainerName(), blobName);
+    public boolean deleteResponseBlob(String blobName) {
```

Comment:
We cant use this method as the list blobs returns the blob name and path. You need to use the deleteBlobData instead

Reply:
It seemed fine for me to use - the parameter contains the full name e.g. `dropzone/DARTS/response/2760_187_1_2d50a0bbde794e0ea9f4918aafeaccde_0_iu.rsp` and they are the response files we are deleting.
But I can rename the parameter to be `blobPathAndName` or choose to rename the operation to be `deleteBlobData` as you have on your DMP-2139-ARMCleanUpJobAsynchronous branch.

But it's only used in my new method `ArmResponseFilesProcessSingleElementImpl#deleteResponseBlobs` anyway.

I'll have a look - but it looks like the interface param for getBlobData probably needs to be changed to match the implementation also

Reply:
Done in commit c2a2233f